include/fty_common_mlm_library.h

```diff
@@ -34,7 +34,6 @@
 #include <malamute.h>
 #include <cxxtools/allocator.h>
 #include <fty_log.h>
-#include <openssl/sha.h>
```

Comment:
This is autogenerated code, so no changes should happen here

Reply:
well, the idea is to find out why tpower is coredumping, and since getting arm image takes ages i hoped to save some time